.github/workflows/pypi-cd.yml

```diff
@@ -7,67 +7,62 @@ on:
 
 jobs:
   build:
+    name: Build wheels on ${{ matrix.os }} for ${{ matrix.arch }}
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macOS-latest, windows-latest]
-        python-version: ["3.7", "3.8", "3.9", "3.10"]
-        node-version: ["14.x"]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        arch: [auto]
         include:
-          - { os: ubuntu-latest, python-version: "3.7", python-abis: "cp37-cp37m" }
-          - { os: ubuntu-latest, python-version: "3.8", python-abis: "cp38-cp38" }
-          - { os: ubuntu-latest, python-version: "3.9", python-abis: "cp39-cp39" }
-          - { os: ubuntu-latest, python-version: "3.10", python-abis: "cp310-cp310" }
-          - { os: windows-latest, python-version: "3.10", build-static: 1 }
+          - os: ubuntu-latest
+            arch: aarch64
+          - os: macos-latest
+            arch: universal2
 
     steps:
       - name: Check out code
         uses: actions/checkout@v2
 
-      - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v2
-        with:
-          node-version: ${{ matrix.node-version }}
+      - name: Set up QEMU
+        if: ${{ matrix.arch == 'aarch64' }}
+        uses: docker/setup-qemu-action@v1
 
-      - name: Set up conda ${{ matrix.python-version }}
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v2.3.1
         env:
-          PYTHON: ${{ matrix.python-version }}
-        shell: bash
-        run: |
-          source ./ci/install-conda.sh
-          python -m pip install --upgrade pip setuptools wheel coverage;
+          CIBW_BEFORE_BUILD: pip install -r ci/requirements-wheel.txt
+          CIBW_ARCHS: ${{ matrix.arch }}
+          CIBW_SKIP: pp* *-musllinux* cp36-* cp310-win32 *i686
```

Comment:
We release packages with `manylinux1` images under x86 for Python < 3.9. This need to be keeped.